integration/controller_test.go

```diff
@@ -256,6 +256,91 @@ var _ = Describe("create", func() {
 				}).Should(WithTransform(getData, Equal(expected)))
 			})
 		})
+
+		Context("With wrong namespace and cluster-wide annotation", func() {
+			var ns2 string
+			BeforeEach(func() {
+				ns2 = createNsOrDie(c, "create")
+				ss.Namespace = ns2
+			})
+			BeforeEach(func() {
+				var err error
+
+				s.Annotations = map[string]string{
+					ssv1alpha1.SealedSecretClusterWideAnnotation: "true",
+				}
+
+				fmt.Fprintf(GinkgoWriter, "Re-sealing secret %#v", s)
+				ss, err = ssv1alpha1.NewSealedSecret(scheme.Codecs, pubKey, s)
```

Comment:
Need to move the `ss.Namespace = ns2` from the `BeforeEach` to just after this line.